ts/lib220.ts

```diff
@@ -72,8 +72,8 @@ export class Line {
 }
 
 function checkIfPoint(p: any) {
-  if (typeof(p) !== 'object' || 
-      typeof(p.x) !== 'number' || 
+  if (typeof(p) !== 'object' ||
```

Comment:
If you're using VSCode, use this in `settings.json`:

```"trailing-spaces.deleteModifiedLinesOnly": true```

Reply:
I use Sublime; I can be more careful of these extraneous stylistic edits.